README.md

```diff
@@ -1,8 +1,8 @@
 # Linear and generalized linear models in Julia
 
-| **Documentation** | **Release status** | **Master status** | Coverage |
+| **Documentation** | **CI status** | Coverage |
 |:-----------------:|:------------------:|:-----------------:|:--------:|
-| [![][docs-stable-img]][docs-stable-url] [![][docs-latest-img]][docs-latest-url] | [![][pkg-0.6-img]][pkg-0.6-url] | [![][travis-img]][travis-url] | [![][coveralls-img]][coveralls-url] |
+| [![][docs-stable-img]][docs-stable-url] [![][docs-latest-img]][docs-latest-url] | [![][travis-img]][travis-url] | [![][coveralls-img]][coveralls-url] |
 
 [docs-latest-img]: https://img.shields.io/badge/docs-latest-blue.svg
 [docs-latest-url]: https://JuliaStats.github.io/GLM.jl/latest
```

Comment:
I think this will need to be changed to `/dev`.

Reply:
The url works for the current docs so is this because of a change to Documenter?

Reply:
Yes AFAIK: https://juliadocs.github.io/Documenter.jl/stable/lib/public/#Documenter.deploydocs